guidelines/index.html

```diff
@@ -443,15 +443,16 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
                					
                
                					
-               <p>In order to avoid confusion for implementers for whom backwards compatibility to WCAG 2 versions is important, new success criteria in WCAG 2.2 have been appended to the end of the set of success criteria within their guideline. This avoids the need to change the section number of success criteria from WCAG 2, which would be caused by inserting new success criteria between existing success criteria in the guideline, but it means success criteria in each guideline are no longer grouped by conformance level. 各ガイドライン内での達成基準の順番は、適合レベルに関する情報を暗示するものではなく、各達成基準自身の適合レベル表示 (A / AA / AAA) のみが適合レベルを示す。<a href="https://www.w3.org/WAI/WCAG22/quickref/">WCAG 2.2 クイックリファレンス</a>は、他の多数のフィルターや並べ替え方法と共に、適合レベルごとにグループ化された達成基準を見る方法を提供する。</p>
+               <p>WCAG 2 系のバージョンに対する後方互換性が重要となる実装者の混乱を招くことを避けるため、WCAG 2.2 における新しい達成基準はガイドライン内の一連の達成基準の末尾に追加されている。これにより、ガイドラインの既存の達成基準の間に新しい達成基準を挿入することによって生じるであろう、WCAG 2 からある達成基準のセクション番号を変更する必要がなくなる。しかしながらこのことは、各ガイドライン内の達成基準がもはや適合レベルごとにグループ化されていないことを意味する。各ガイドライン内での達成基準の順番は、適合レベルに関する情報を暗示するものではなく、各達成基準自身の適合レベル表示 (A / AA / AAA) のみが適合レベルを示す。<a href="https://www.w3.org/WAI/WCAG22/quickref/">WCAG 2.2 Quick Reference</a> が、他の多数のフィルター及び並べ替え方法とともに、適合レベルごとにグループ化された達成基準を見る方法を提供するだろう。</p>
```

Comment:
「後方互換性が重要となる実装者の混乱を招くことを避ける」は「重要となる」がどこにかかるかわかりにくいように思いました (「重要となる実装者」「重要となる混乱」のように読める)。「後方互換性を重視する」というくらいでどうでしょうか。「混乱を招くことを避ける」は単に「混乱を避ける」で良いかも知れません (原文 avoid confusion なので)。

```suggestion
               <p>WCAG 2 系のバージョンに対する後方互換性を重視する実装者の混乱を避けるため、WCAG 2.2 における新しい達成基準はガイドライン内の一連の達成基準の末尾に追加されている。これにより、ガイドラインの既存の達成基準の間に新しい達成基準を挿入することによって生じるであろう、WCAG 2 からある達成基準のセクション番号を変更する必要がなくなる。しかしながらこのことは、各ガイドライン内の達成基準がもはや適合レベルごとにグループ化されていないことを意味する。各ガイドライン内での達成基準の順番は、適合レベルに関する情報を暗示するものではなく、各達成基準自身の適合レベル表示 (A / AA / AAA) のみが適合レベルを示す。<a href="https://www.w3.org/WAI/WCAG22/quickref/">WCAG 2.2 Quick Reference</a> が、他の多数のフィルター及び並べ替え方法とともに、適合レベルごとにグループ化された達成基準を見る方法を提供するだろう。</p>
```

Reply:
鮮やかで痺れました！！